test/orchard/pp/pp_array_test.clj

```diff
@@ -0,0 +1,16 @@
+(ns orchard.pp.pp-array-test
```

Comment:
I don't think we need all pp tests in Orchard. Just 1-2 sanity checks for `orchard.print/pprint-str` is sufficient.

Reply:
I removed most of the pp test files and combined some of the tests in `pp_test.clj` so we have some coverage for the `pp.clj` file as well. If you want me to remove those tests as well, let me know.

I also added 2 tests for `orchard.print/pprint-str`, but I have trouble getting them to run in Leiningen. For some reason the tests are passing in the REPL, but fail when I run them in Leiningen. atoms for example are printed differently. I mentioned this in the TODO section above. Any idea why that is?